internal/flags/flags.go

```diff
@@ -33,45 +33,50 @@ type Flags struct {
 
 func buildFlags(args *Args) []cli.Flag {
 	return []cli.Flag{
-		&cli.BoolFlag{
-			Name:        "version",
-			Usage:       "print version and build information",
-			Destination: &args.PrintVersion,
-		},
 		&cli.StringFlag{
 			Name:        "root",
-			Usage:       "the directory storing snapshotter working states",
+			Usage:       "directory to store snapshotter data and working states",
 			Destination: &args.RootDir,
 		},
 		&cli.StringFlag{
 			Name:        "address",
-			Usage:       "gRPC socket path",
+			Usage:       "remote snapshotter gRPC socket path",
 			Destination: &args.Address,
 		},
 		&cli.StringFlag{
 			Name:        "config",
-			Usage:       "path to the nydus-snapshotter configuration",
+			Usage:       "path to nydus-snapshotter configuration file",
 			Destination: &args.SnapshotterConfigPath,
 		},
+		&cli.StringFlag{
+			Name:        "nydus-image",
+			Usage:       "path to `nydus-image` binary, default to search in $PATH",
+			Destination: &args.NydusImagePath,
+		},
+		&cli.StringFlag{
+			Name:        "nydusd",
+			Usage:       "path to `nydusd` binary, default to search in $PATH",
+			Destination: &args.NydusdPath,
+		},
 		&cli.StringFlag{
 			Name:        "nydusd-config",
 			Aliases:     []string{"config-path"},
-			Usage:       "path to the nydusd configuration",
+			Usage:       "path to nydusd configuration file",
 			Destination: &args.NydusdConfigPath,
 		},
 		&cli.StringFlag{
 			Name:        "daemon-mode",
-			Usage:       "spawning nydusd daemon mode, legal values include \"multiple\", \"shared\" or \"none\"",
+			Usage:       "nydusd daemon working mode, possible values: \"dedicated\"/\"multiple\", \"shared\" or \"none\"",
```

Comment:
`dedicated` should not be included in this PR?

Reply:
good catch